test/models/work_test.rb

```diff
@@ -0,0 +1,76 @@
+require "test_helper"
+
+describe Work do
+  let(:work) { Work.new }
```

Comment:
Similar to the `upvote` tests above make the work valid and then take a way a specific field to make it invalid and then test that.